src/senaite/sync/fieldmanagers.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017-2017 SENAITE LIMS.
+
+from zope import interface
+
+from senaite.jsonapi.fieldmanagers import ATFieldManager
+from senaite.jsonapi.interfaces import IFieldManager
+
+
+class StringFieldManager(ATFieldManager):
+    """Adapter to set/get the value of String Fields
+    """
+    interface.implements(IFieldManager)
+
+    def set(self, instance, value, **kw):
+        """Set the value of the field
+        """
+        if value is None:
+            if self.field.multiValued:
```

Comment:
I think this is not needed with the UIDReferenceField manager in place. But I'm ok to leave this as it is when you want.

Reply:
@ramonski I would leave it as it is by now since Nihad needs this field manager and we can remove it later when the UIDReferenceField manager is in place.